proto/cacheclient.proto

```diff
@@ -455,18 +455,45 @@ message _SortedSetFetchRequest {
     DESCENDING = 1;
   }
 
+  // num_results fields
   message _All {}
 
   message _Limit {
     uint32 limit = 1;
   }
 
+  // range fields
+  message _ByIndex {}
```

Comment:
Should this also have structures similar to endpoints?

Reply:
What's already implemented is "ByIndex", and it doesn't require additional arguments.